quickstarts/flutter-android/config.yml

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
can remove the top empty lines

Reply:
Hi @pkosanam,
Removed empty lines. Kindly review it.

Thank you...